src/service/performance-impl.js

```diff
@@ -473,6 +491,14 @@ export class Performance {
       });
     }
 
+    if (this.supportsEvents_) {
+      this.createPerformanceObserver_(processEntry, {
+        type: 'event',
+        durationThreshold: 16, // Minimim duration of 16ms, as provided by the spec
```

Comment:
would you mind providing/explaining where we got this value?

I'm looking at the implementation on the web-vitals library and they seem to set this to 40
https://github.com/GoogleChrome/web-vitals/blob/main/src/onINP.ts#L132C74-L133C30

Reply:
I saw this from another implementation. web-vital's rational for 40ms makes sense, and I can update it.